components/FeaturedSection.js

```diff
@@ -0,0 +1,90 @@
+import { useContext, useState } from "react";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Carousel from "react-bootstrap/Carousel";
+import { useMediaQuery } from "react-responsive";
+import { IoIosArrowDropleft, IoIosArrowDropright } from "react-icons/io";
+import IssuesContext from "../contexts/IssuesContext";
+import IndexIssueListItem from "./shared/IndexIssueListItem";
+
+export default function FeaturedSection() {
+  // Set state for next/previous arrows so they disappear on slide
+  const [showArrows, setShowArrows] = useState(true);
+
+  const { issues } = useContext(IssuesContext);
+  const featuredIssues = issues.filter((issue) => issue.isFeatured);
+
+  const isTabletOrMobile = useMediaQuery({ query: "(max-width: 996px)" });
+
+  return (
+    <Row>
+      {/* Render rows with featured cards if tablet or mobile else render carousel */}
+      {isTabletOrMobile ? (
+        <Col className="mx-5">
+          <Row className="pt-3 pb-4 text-primary text-center">
+            <Col>
+              <h4 className="fw-bold">Featured</h4>
+            </Col>
+          </Row>
+          <Row>
+            {/* This slice is incase there are more than 8 issues with the featured label */}
+            {featuredIssues.slice(0, 8).map((issue) => {
```

Comment:
We always want to have just 8 otherwise the design is thrown off so this is a safeguard in case an extra featured label is added to an issue